java/extensions/semantickernel-settings-loader/src/main/java/com/microsoft/semantickernel/connectors/ai/openai/util/AzureOpenAISettings.java

```diff
@@ -7,9 +7,9 @@
 
 /** Settings for Azure OpenAI client */
 public class AzureOpenAISettings extends AbstractOpenAIClientSettings {
-    private static final String DEFAULT_SETTINGS_PREFIX = "client.azureopenai";
-    private static final String AZURE_OPEN_AI_ENDPOINT_SUFFIX = "endpoint";
-    private static final String AZURE_OPEN_AI_DEPLOYMENT_NAME_SUFFIX = "deploymentname";
+    public static final String DEFAULT_SETTINGS_PREFIX = "client.azureopenai";
```

Comment:
My recommendation here is to have a static getter method for these. The issue is, if you reference DEFAULT_SETTINGS_PREFIX in your code, the compiler will do a straight-up string replacement. If the connectors library later changes the value, your code is not going to work with the newer library. If you use a getter, your code will always have the correct value from the connectors library.  

e.g., `public static final String getDefaultSettingsPrefix() { return DEFAULT_SETTINGS_PREFIX; }`